src/main/java/frc/robot/subsystems/lights/LightsConstants.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.subsystems.lights;
+
+import frc.robot.Constants;
+
+public class LightsConstants {
+  public static final int NUM_CANDLE_LEDS = Constants.PLACEHOLDER_INT;
+  public static final int LIGHT_BRIGHTNESS = 1;
```

Comment:
I could be wrong here but I looked over the documentation and "brightness" is also a double. therefore, LIGHT_BRIGHTNESS should be a double with a value of 1.0. I think that having it as an int also works but generally we like to make it a double and add the .0 to be more clear
<img width="565" alt="Screenshot 2024-02-12 at 7 20 50 PM" src="https://github.com/cachemoney8096/2024-crescendo/assets/109251157/a157859e-5c25-42c6-b2b8-67125f7aa8bb">

Reply:
Yes, brightness should be a double for animations (it's [0,1], so if it was an int it wouldn't really be brightness and it would just be on/off). An integer will work (Java can typecast an integer n to a double n.0) but it lacks clarity and there really isn't a reason to have it as an int here so this should be a double.